backend/src/db-utils.ts

```diff
@@ -205,7 +205,9 @@ const queryStakingPoolAccountIds = async (): Promise<string[]> => {
       [
         `SELECT account_id as "accountId"
     FROM accounts
-    WHERE account_id LIKE '%${nearStakingPoolAccountSuffix}'`,
+    WHERE account_id LIKE '%${
+      config.accountIdSuffix.stakingPool[config.networkName]
+    }'`,
```

Comment:
Will this query works normally in case of 'localhostnet'?

Reply:
The line will be `WHERE account_id LIKE '%undefined'` and will return zero rows.
Looks good enough to me :)